src/GitVersionCore/packages.config

```diff
@@ -1,10 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Caseless.Fody" version="1.3.7" targetFramework="net40" developmentDependency="true" />
-  <package id="Fody" version="1.29.3" targetFramework="net40" developmentDependency="true" />
-  <package id="JetBrainsAnnotations.Fody" version="1.0.4.0" targetFramework="net40" developmentDependency="true" />
-  <package id="LibGit2Sharp" version="0.21.0.176" targetFramework="net40" />
-  <package id="PepitaPackage" version="1.21.4" targetFramework="net40" developmentDependency="true" />
-  <package id="Visualize.Fody" version="0.4.4.0" targetFramework="net40" developmentDependency="true" />
-  <package id="YamlDotNet" version="3.6.1" targetFramework="net40" />
+  <package id="Caseless.Fody" version="1.3.7" targetFramework="net4" developmentDependency="true" />
+  <package id="Fody" version="1.29.3" targetFramework="net4" developmentDependency="true" />
+  <package id="JetBrainsAnnotations.Fody" version="1.0.4.0" targetFramework="net4" developmentDependency="true" />
+  <package id="LibGit2Sharp" version="0.22.0-pre20150716071016" targetFramework="net4" />
+  <package id="LibGit2Sharp.NativeBinaries" version="1.0.72" targetFramework="net4" />
+  <package id="PepitaPackage" version="1.21.4" targetFramework="net4" developmentDependency="true" />
```

Comment:
Newer version of nuget is changing it automatically. A bit annoying really

Sent from my Windows Phone
________________________________
From: nulltoken<mailto:notifications@github.com>
Sent: ‎10/‎09/‎2015 6:34 PM
To: GitTools/GitVersion<mailto:GitVersion@noreply.github.com>
Subject: Re: [GitVersion] Fix for mono support in GitVersionTask (#638)


In src/GitVersionCore/packages.config<https://github.com/GitTools/GitVersion/pull/638#discussion_r39145670>:

> @@ -1,10 +1,11 @@
>  ?<?xml version="1.0" encoding="utf-8"?>
>  <packages>
> -  <package id="Caseless.Fody" version="1.3.7" targetFramework="net40" developmentDependency="true" />
> -  <package id="Fody" version="1.29.3" targetFramework="net40" developmentDependency="true" />
> -  <package id="JetBrainsAnnotations.Fody" version="1.0.4.0" targetFramework="net40" developmentDependency="true" />
> -  <package id="LibGit2Sharp" version="0.21.0.176" targetFramework="net40" />
> -  <package id="PepitaPackage" version="1.21.4" targetFramework="net40" developmentDependency="true" />
> -  <package id="Visualize.Fody" version="0.4.4.0" targetFramework="net40" developmentDependency="true" />
> -  <package id="YamlDotNet" version="3.6.1" targetFramework="net40" />
> +  <package id="Caseless.Fody" version="1.3.7" targetFramework="net4" developmentDependency="true" />
> +  <package id="Fody" version="1.29.3" targetFramework="net4" developmentDependency="true" />
> +  <package id="JetBrainsAnnotations.Fody" version="1.0.4.0" targetFramework="net4" developmentDependency="true" />
> +  <package id="LibGit2Sharp" version="0.22.0-pre20150716071016" targetFramework="net4" />
> +  <package id="LibGit2Sharp.NativeBinaries" version="1.0.72" targetFramework="net4" />
> +  <package id="PepitaPackage" version="1.21.4" targetFramework="net4" developmentDependency="true" />


Why do we change the targetFramework attribute from net40 to net4 here?

—
Reply to this email directly or view it on GitHub<https://github.com/GitTools/GitVersion/pull/638/files#r39145670>.